<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormBaixaReceber</class>
 <widget class="QWidget" name="FormBaixaReceber">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1313</width>
    <height>815</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RECEBIMENTO</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#FormBaixaReceber{
	background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:1, stop:0 rgba(0, 0, 0, 118), stop:1 rgba(255, 255, 255, 255));

}

QLabel{
font-size: 12px;
font-family: &quot;Arial&quot;;
font-weight: bold;
color: #277298;
border: none;
}
QLineEdit{
background-color: rgb(255, 255, 127);
font-size: 24px;
font-family: &quot;Arial&quot;;
font-weight: bold;
color: rgb(0, 0, 0);
border:none;
border-bottom-left-radius:0px;
border-bottom-right-radius:0px;
border-top-left-radius:0px;
}
QLineEdit:focus{
	background-color: rgb(255, 255, 255);
}
QComboBox{
border:1px solid;
border-bottom-left-radius:0px;
border-bottom-right-radius:0px;
border-top-left-radius:0px;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="styleSheet">
      <string notr="true">
QWidget#widget_2{
	background-color: rgb(0, 0, 0,0);
text-transform: uppercase;

}
</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>1000</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>800</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget{
text-transform: uppercase;
border:1px solid blue;
	background-color: rgb(231, 231, 231);
}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="QFrame" name="frame">
           <property name="styleSheet">
            <string notr="true">QFrame{
border:none;
}</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QFrame" name="frame_4">
              <property name="font">
               <font>
                <pointsize>24</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QFrame#frame_4{
color:red;
border: none;
background-color:#2408D4;}

QLabel{color: white;
background-color:transparent;
font: 75 15pt &quot;MS Shell Dlg 2&quot;;

}
</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label">
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="text">
                  <string>caixa-receber</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_3">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>220</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QFrame{border:none;

}
QGroupBox{
border:none;
}
</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QFrame" name="frame_5">
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_2">
                  <item>
                   <widget class="QLabel" name="label_6">
                    <property name="text">
                     <string>Tipo PAgamento</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="cb_tipoDocumento">
                    <property name="minimumSize">
                     <size>
                      <width>150</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <pointsize>16</pointsize>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QComboBox{
padding:10px;
background-color: rgb(255, 255, 255);
}

QComboBox:focus {
    border: 2px solid red;
	background-color: rgb(255, 255, 127);
	color: rgb(0, 0, 0);
}
</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="verticalSpacer_2">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>20</width>
                      <height>40</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QFrame" name="frame_6">
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_5">
                  <item>
                   <widget class="QLabel" name="label_2">
                    <property name="text">
                     <string>data atual</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QDateEdit" name="dt_datapamento">
                    <property name="focusPolicy">
                     <enum>Qt::ClickFocus</enum>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QDateEdit {
	border: 2px solid lightgray;
	border-radius: 5px;
	font: 20pt &quot;MS Shell Dlg 2&quot;;
	background-color: rgb(255, 255, 255);
}
</string>
                    </property>
                    <property name="calendarPopup">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="verticalSpacer">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>20</width>
                      <height>40</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QFrame" name="frame_7">
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout">
                  <item>
                   <widget class="QLabel" name="label_4">
                    <property name="text">
                     <string>valor total </string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QDoubleSpinBox" name="db_valorTotalparcela">
                    <property name="minimumSize">
                     <size>
                      <width>300</width>
                      <height>40</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">background-color: rgb(255, 255, 127);
font-size: 24px;
font-family: &quot;Arial&quot;;
font-weight: bold;
color: rgb(0, 0, 0);
border:none;
border-bottom-left-radius:0px;
border-bottom-right-radius:0px;
border-top-left-radius:0px;</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                    </property>
                    <property name="buttonSymbols">
                     <enum>QAbstractSpinBox::NoButtons</enum>
                    </property>
                    <property name="showGroupSeparator" stdset="0">
                     <bool>true</bool>
                    </property>
                    <property name="decimals">
                     <number>6</number>
                    </property>
                    <property name="maximum">
                     <double>999999.000000000000000</double>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_3">
                    <property name="text">
                     <string>valor desconto %</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QDoubleSpinBox" name="Db_desconto">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>40</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">background-color: rgb(255, 255, 127);
font-size: 24px;
font-family: &quot;Arial&quot;;
font-weight: bold;
color: rgb(0, 0, 0);
border:none;
border-bottom-left-radius:0px;
border-bottom-right-radius:0px;
border-top-left-radius:0px;</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                    </property>
                    <property name="buttonSymbols">
                     <enum>QAbstractSpinBox::NoButtons</enum>
                    </property>
                    <property name="showGroupSeparator" stdset="0">
                     <bool>true</bool>
                    </property>
                    <property name="prefix">
                     <string>DESCONTO% </string>
                    </property>
                    <property name="decimals">
                     <number>3</number>
                    </property>
                    <property name="maximum">
                     <double>100.000000000000000</double>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_5">
                    <property name="text">
                     <string>valor Final</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QDoubleSpinBox" name="db_valortotal">
                    <property name="minimumSize">
                     <size>
                      <width>300</width>
                      <height>40</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">background-color: rgb(255, 255, 127);
font-size: 24px;
font-family: &quot;Arial&quot;;
font-weight: bold;
color: rgb(0, 0, 0);
border:none;
border-bottom-left-radius:0px;
border-bottom-right-radius:0px;
border-top-left-radius:0px;</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                    </property>
                    <property name="buttonSymbols">
                     <enum>QAbstractSpinBox::NoButtons</enum>
                    </property>
                    <property name="showGroupSeparator" stdset="0">
                     <bool>true</bool>
                    </property>
                    <property name="decimals">
                     <number>6</number>
                    </property>
                    <property name="maximum">
                     <double>9999999.000000000000000</double>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="tab_BaixaReceber">
              <property name="focusPolicy">
               <enum>Qt::ClickFocus</enum>
              </property>
              <property name="styleSheet">
               <string notr="true">QTableWidget {
  background-color: white;
	font: 12pt &quot;MS Shell Dlg 2&quot;;
}

QTableWidget QHeaderView::section {
  background-color: gray;
  color: white;
  font-weight: bold;
}

QTableWidget::item {
  border: 0px solid black;
border-bottom:1px solid;

}

QTableWidget::item:selected {
  background-color: blue;
  color: white;
}
</string>
              </property>
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <property name="tabKeyNavigation">
               <bool>false</bool>
              </property>
              <property name="showDropIndicator" stdset="0">
               <bool>true</bool>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
              <attribute name="horizontalHeaderDefaultSectionSize">
               <number>210</number>
              </attribute>
              <attribute name="horizontalHeaderHighlightSections">
               <bool>true</bool>
              </attribute>
              <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
               <bool>true</bool>
              </attribute>
              <attribute name="horizontalHeaderStretchLastSection">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderHighlightSections">
               <bool>true</bool>
              </attribute>
              <column>
               <property name="text">
                <string>id</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>NOTA</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>NOME CLIENTE</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>desconto</string>
               </property>
               <property name="foreground">
                <brush brushstyle="NoBrush">
                 <color alpha="255">
                  <red>255</red>
                  <green>0</green>
                  <blue>0</blue>
                 </color>
                </brush>
               </property>
              </column>
              <column>
               <property name="text">
                <string>valor nominal</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>valor total</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>data emissao</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>data baixa</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>documento</string>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_2">
              <property name="styleSheet">
               <string notr="true">QPushButton{
	text-align:center;
	color:#fff;
	border:0px;
	
	font-size:20px;
	cursor: pointer;
	background-color:#538fff;
padding:15px 15px;
}
QPushButton:hover{
background-color:#3668ff;
}</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <property name="spacing">
                <number>2</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QPushButton" name="bt_receber">
                 <property name="focusPolicy">
                  <enum>Qt::NoFocus</enum>
                 </property>
                 <property name="text">
                  <string>baixar receber (F2)</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>:/telainicial/telainicial/receber.png</normaloff>:/telainicial/telainicial/receber.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>30</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="shortcut">
                  <string>F2</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="bt_cancelar">
                 <property name="focusPolicy">
                  <enum>Qt::NoFocus</enum>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(255, 0, 127);</string>
                 </property>
                 <property name="text">
                  <string>cancelar (esc)</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>:/newPrefix/icons/action_stop.gif</normaloff>:/newPrefix/icons/action_stop.gif</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>30</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="shortcut">
                  <string>Esc</string>
                 </property>
                 <property name="autoRepeatInterval">
                  <number>4</number>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cb_tipoDocumento</tabstop>
  <tabstop>db_valorTotalparcela</tabstop>
  <tabstop>Db_desconto</tabstop>
  <tabstop>db_valortotal</tabstop>
  <tabstop>tab_BaixaReceber</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>bt_cancelar</sender>
   <signal>clicked()</signal>
   <receiver>FormBaixaReceber</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>651</x>
     <y>490</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
